supertokens_python/ingredients/emaildelivery/service/smtp.py

```diff
@@ -13,30 +13,31 @@
 # under the License.
 
 
-from ..types import EmailDeliveryInterface
+import smtplib
 from abc import ABC, abstractmethod
-from typing import Any, Callable, Dict, Generic, TypedDict, TypeVar, Union
+from typing import Any, Callable, Dict, Generic, TypeVar, Union
+
+from ..types import EmailDeliveryInterface
 
 _T = TypeVar('_T')
 
 
 class SMTPServiceConfigAuth:
-    user: str
-    password: str
+    def __init__(self, user: str, password: str) -> None:
+        self.user = user
+        self.password = password
 
 
 class SMTPServiceConfigFrom:
-    name: str
-    email: str
+    def __init__(self, name: str, email: str) -> None:
+        self.name = name
+        self.email = email
 
 
 class SMTPServiceConfig:
     def __init__(
-        self,
-        host: str,
-        email_from: SMTPServiceConfigFrom,
-        port: int,
-        secure: Union[bool, None] = None,
+        self, host: str, email_from: SMTPServiceConfigFrom,
```

Comment:
change `email_from` to `from`

Reply:
We can't. `from` is a reserved keyword :')

Reply:
ohh lol, yeah